test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
We can leave this as is for now. I'd love to add [rspec](https://github.com/rspec/rspec-rails) sometime.